content/ngf/install/manifests/open-source.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Use Manifests to install NGINX Gateway Fabric with NGINX Open Source
+linkTitle: NGINX Open Source
+weight: 100
+toc: true
+nd-content-type: how-to
+nd-product: FABRIC
+nd-docs: DOCS-1429
+---
+
+This page describes how to use Manifests to install NGINX Gateway Fabric with NGINX Open Source.
+
+It explains how to install the Gateway API resources and add certificates for secure authentication, then deploy NGINX Gateway Fabric and its CRDs (Custom resource definitions).
+
+By following these instructions, you will finish with a functional NGINX Gateway Fabric instance for your Kubernetes cluster.
+
+{{< call-out "note" >}} To learn which Gateway API resources NGINX Gateway Fabric currently supports, view the [Gateway API Compatibility]({{< ref "/ngf/overview/gateway-api-compatibility.md" >}}) topic. {{< /call-out >}}
+
+## Before you begin
+
+To complete this guide, you will need the following pre-requisites:
+
+- [A supported Kubernetes version]({{< ref "/ngf/overview/technical-specifications.md" >}})
+- A functional Kubernetes cluster
+- [cert-manager](https://cert-manager.io/docs/installation/)
```

Comment:
cert manager has to be installed in a specific way to support Gateway API, which is what we previously included in our docs

Reply:
Reading the cert-manager documentation has actually caused me some confusion:

> Since cert-manager 1.15, the Gateway API support is no longer gated behind a feature flag, but you still need to enable the Gateway API support.

I'm trying to grok if the separate steps to "enable the Gateway API support" are necessary, and what they might be for Manifests, since the docs are completely prioritizing Helm.

If it turns out cert-manager just "works" now (Which it seemed to be from a deployment perspective -  I haven't tried any of the nuance use cases), then we simply add a note to either use cert-manager 1.15+ or manually enable the API for earlier releases.